tests/test_goodconf.py

```diff
@@ -141,17 +141,17 @@ class TestConf(GoodConf):
         a: bool = Field(default=False)
 
     lines = TestConf.generate_markdown().splitlines()
-    assert "  * type: `bool`" in lines
+    assert "  * type: `<class 'bool'>`" in lines
```

Comment:
Is it hard to get it to maintain the original output? (no `<class '...'>`)

Reply:
It's possible to do!